### PR TITLE
User Profile page & Readme

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR
- Removing redundant `fakeData.js`
- Update readme
- Adding User Profile page
user page

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Clean and readable code, the abstractions on auth part are great. Good to merge

File: src/pages/User.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          </ul>
+          <ul className="list-content">
+            <li>{userData.firstName}</li>
+            <li>{userData.lastName}</li>
+            <li>{userData.email}</li>
+            <li>
+              {togglePass
+                ? userData.password
+                : [...userData.password].map(() => "*")}
+              <div className="eye-btn">
```
Comment (bhushannaikaroskar) [inline src/pages/User.jsx]: You can avoid using destructuring here as map return new array so you may use userData.password.map(() => "*") .

**Review by bhushannaikaroskar — APPROVED** [review]
Hey sudip good work on writing README file. Loved the way you have explained installation part  as well. A minor suggestion from my side please do check before merging. Rest I think your code is great.

File: src/pages/User.jsx (hunk 1 above)
Comment (sudipkundu999) [reply src/pages/User.jsx]: `userData.password` is a string, not an array. 
So, by destructuring I'm spreading the string into an array.

Comment (sudipkundu999) [reply src/pages/User.jsx]: I'll just use `split('')` here.

**Pull request merged**